cloudinit/set-falcon-sensor-customer-id.py

```diff
@@ -61,27 +59,40 @@ def main() -> int:
     )
 
     # Get the values of the SSM Parameter Store parameters
-    nessus_hostname: str = get_parameter(ssm_client, NESSUS_HOSTNAME_KEY)
-    nessus_key: str = get_parameter(ssm_client, NESSUS_KEY_KEY)
-    nessus_port: str = get_parameter(ssm_client, NESSUS_PORT_KEY)
-
-    # Link the Nessus Agent
-    link_cmd: list[str] = [
-        f"{NESSUS_AGENT_INSTALL_PATH}/sbin/nessuscli",
-        "agent",
-        "link",
-        f"--key={nessus_key}",
-        f"--host={nessus_hostname}",
-        f"--port={nessus_port}",
-        f"--groups={NESSUS_GROUPS}",
+    customer_id: str = get_parameter(ssm_client, FALCON_CUSTOMER_ID_KEY)
+
+    #
+    # Set the customer ID
+    #
+    customer_id_cmd: list[str] = [
+        f"{FALCON_SENSOR_INSTALL_PATH}/falconctl",
+        "-s",
+        f"--cid={customer_id}",
     ]
     # Bandit triggers B603 here, but we're using subprocess.run()
-    # safely here, since the variable content in link_cmd comes
+    # safely here, since the variable content in customer_id__cmd
+    # comes directly from SSM Parameter Store.  For more details on
+    # B603 see here:
+    # https://bandit.readthedocs.io/en/latest/plugins/b603_subprocess_without_shell_equals_true.html
+    cid_cp: subprocess.CompletedProcess = subprocess.run(customer_id_cmd)  # nosec
+    if cid_cp.returncode != 0:
+        return cid_cp.returncode
+
+    #
+    # Restart the Falcon sensor
+    #
+    restart_cmd: list[str] = [
+        "/usr/bin/systemctl",
+        "restart",
+        "falcon-sensor.service",
+    ]
+    # Bandit triggers B603 here, but we're using subprocess.run()
+    # safely here, since the variable content in restart_cmd comes
     # directly from SSM Parameter Store.  For more details on B603 see
```

Comment:
Nothing from SSM is used in this second command. Please update this comment to reflect what is in the `restart_cmd` list.

Reply:
Please see commit 3c890ea.